pymc3/step_methods/hmc/nuts.py

```diff
@@ -69,39 +74,38 @@ def astep(self, q0):
         else:
             step_size = np.exp(self.log_step_size_bar)
 
-        u = floatX(nr.uniform())
+        u = np.typeDict[theano.config.floatX](nr.uniform())
```

Comment:
hmm... could have sworn there was a reasonable speedup from this, but can't see it any more.  I'll revert these two (there's another in the `buildtree`).